articles/evaluation_queues.md

```diff
@@ -0,0 +1,164 @@
+
+<br/>
+
+{% include note.html content="The name of your evaluation queue MUST be unique, otherwise the queue will not be created." %}
+
+The example below shows how to create a queue using all of the parameters described:
+ 
+
+{% tabs %}
+{% tab Python %}
+{% highlight python %}
+import synapseclient
+
+syn = synapseclient.login()
+
+evaluation = synapseclient.Evaluation(name="My Unique Example Challenge Name",
+	description="Short description of challenge queue",
+	status="OPEN",
+	contentSource="syn12345", #Your Synapse Project synID
+	submissionInstructionsMessage="Instructions on submission format...",
+	submissionReceiptMessage="Thanks for submitting to My Example Challenge!",
+	quota={'submissionLimit':3, #The maximum number of submissions per team/participant per round.
+		'firstRoundStart':'2017-11-02T07:00:00.000Z', #The date/time ("%Y-%m-%dT%H:%M:%S%Z") at which the first round begins in UTC. 
+		'roundDurationMillis':1645199000, #The duration of each round.
+		'numberOfRounds':1} #	The number of rounds, or null if there is no end. (Based on the duration of each round) 
+)
+
+syn.store(evaluation)
+{% endhighlight %}
+{% endtab %}
+
+{% tab R %}
+{% highlight r %}
+library(synapser)
+
+synLogin()
+
+evaluation <- Evaluation(name="My Unique Example Challenge Name",
+	description="Short description of challenge queue",
+	status="OPEN",
+	contentSource="syn12345", #Your Synapse Project synID
+	submissionInstructionsMessage="Instructions on submission format...",
+	submissionReceiptMessage="Thanks for submitting to My Example Challenge!",
+	quota=c(submissionLimit=3, #The maximum number of submissions per team/participant per round.
+    		firstRoundStart = '2017-11-02T07:00:00.000Z', #The date/time ("%Y-%m-%dT%H:%M:%S%Z") at which the first round begins in UTC.
+    		roundDurationMillis = 1645199000, #The duration of each round.
+    		numberOfRounds=1) #	The number of rounds, or null if there is no end. (Based on the duration of each round)
+    )
+    	
+synStore(evaluation)
+		{%endhighlight %}
+{% endtab %}
+
+{% tab Web %}
+You can create Evaluation queues on the web by navigating to your challenge site by adding `/admin` to the url (E.g. www.synapse.org/#!Synapse:syn12345/admin).  Click **Tools** on the right corner and **Add Evaluation Queue** and follow the prompts.
+
+<img src="/assets/images/create_evaluation_queues.png">
+
+{% endtab %}
+{% endtabs %}
+
+## Configure an Evaluation Queue 
+
+Submission "rounds" (start date, round duration, and number of rounds) with an optional submission quota (maximum submissions per participant or team) can be defined for each queue.  There is not a way to configure the round or quota settings of an evaluation queue from the web. The Evaluation ID can be found under the **Challenge** tab of your project. Please note that a Challenge tab will not appear on your project until you have created a challenge (**Tools->Run Challenge**). In the case below, the evaluation queue id is `9610091`.  
```

Comment:
sentence is confusing - separate concepts of rounds from quotas.

Reply:
"There is no way to configure..."

Reply:
"...settings of an Evaluation queue..."

Reply:
Fixed.